assignment/impl/src/java/org/sakaiproject/assignment/impl/AssignmentServiceImpl.java

```diff
@@ -2983,6 +2983,12 @@ public String getMaxPointGradeDisplay(int factor, int maxGradePoint) {
         return getGradeDisplay(Integer.toString(maxGradePoint), Assignment.GradeType.SCORE_GRADE_TYPE, factor);
     }
 
+    public String getFactoredGrade(int factor, String grade, String delimiter) {
```

Comment:
What's the purpose of this method? What's the difference with 'getGradeDisplay'?

getGradeDisplay (grade, Assignment.GradeType.SCORE_GRADE_TYPE, factor)